<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labbe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>      scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>      area</string>
    </property>
   </widget>
   <widget class="QLabel" name="labbe_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>perimetre</string>
    </property>
   </widget>
   <widget class="QLabel" name="la">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>420</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="be">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>450</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ann">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate angle</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="change_scale">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>420</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="change_x">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-200.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="change_y">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-200.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="change_angle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1080.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1080.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="area">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>450</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="perimetre">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>480</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="change_figure">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rhombus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right polygon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Star</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>